uiautomationutilities/perf_tool/readme.md

```diff
@@ -8,148 +8,106 @@
-- 1. Take a checkout of C# tool repo (Details above)
-- 2. Open the Location of the perf tool (Location details given above) in Visual studio.
-- 3. Build Solution by clicking on the Build -> Build Solution. Make sure Release build is chosen for deployment.
-- 4. Bundle following files into one zip file named "PerfRunnables.zip":
+- Take a checkout of C# tool repo (Details above)
+- Open the Location of the perf tool (Location details given above) in Visual studio.
+- Build Solution by clicking on the Build -> Build Solution. Make sure Release build is chosen for deployment.
+- Bundle following files into one zip file named "PerfRunnables.zip":
 	- a. Files "PerfIdentity.exe", "PerfIdentity.runtimeconfig.json", "PerfIdentity.dll" and "PerfIdentity.deps.json" from location  "bin\Release\netcoreapp3.1" and file "PerfDataConfiguration.xml" from root location of the project
-- 5. Go to the pipeline and then from the left side options, Open Library page and upload the file "PerfRunnables.zip" after deleting previous one (if exists)
-- 6. Go to the pipeline definition and make change in the task "Download Perf runnable package" by selecting the file name which was uploaded in the "secure file" drop down.
-- 7. Save the build definition. 
+- Go to the pipeline and then from the left side options, Open Library page and upload the file "PerfRunnables.zip" after deleting previous one (if exists)
+- Go to the pipeline definition and make change in the task "Download Perf runnable package" by selecting the file name which was uploaded in the "secure file" drop down.
+- Save the build definition. 
 
 
 ### Steps to baseline a build number:
-- 1. Go to the pipeline page where we can see the previous runs.
-- 2. From the list of the runs, Click the particular run which we want to choose as the baseline.
-- 3. From the URL, find the numberic value of the buildId and note it. For example buildId will be 708340 for following URL:
-	https://dev.azure.com/IdentityDivision/IDDP/_build/results?buildId=708340&view=results
-- 4. Go to the run page of the pipeline again.
-- 5. Go to the definition of the pipeline by clicking on the "Edit" button on the top right.
-- 6. Click on variables.
-- 7. Change the value of the variable baseBuildID with the value we have noted in the steps above.
-- 8. Save the pipeline definition.
+- Go to the pipeline page where we can see the previous runs.
+- From the list of the runs, Click the particular run which we want to choose as the baseline.
+- From the URL, find the numberic value of the buildId and note it. For example buildId will be 708340 for following URL: https://dev.azure.com/IdentityDivision/IDDP/_build/results?buildId=708340&view=results
+- Go to the run page of the pipeline again.
+- Go to the definition of the pipeline by clicking on the "Edit" button on the top right.
+- Click on variables.
+- Change the value of the variable baseBuildID with the value we have noted in the steps above.
```

Comment:
Suggest clarifying which value `baseBuildID` refers to.... is it the `scenario` code?

Reply:
`baseBuildID ` is now part of the definitions at the start of the doc